<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newAssetUI</class>
 <widget class="QDialog" name="newAssetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>269</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>269</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>269</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New Asset</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(35,35,35)</string>
     </property>
     <property name="text">
      <string>Create New Asset</string>
     </property>
     <property name="margin">
      <number>3</number>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="cancelBtn">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Rig Types</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="animChx">
        <property name="text">
         <string>Rig</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="gameChx">
        <property name="text">
         <string>Game Rig</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mocapChx">
        <property name="text">
         <string>Mocap Rig</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="otherChx">
          <property name="text">
           <string>Other</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="otherLine">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="createBtn">
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Asset Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLine"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>otherChx</sender>
   <signal>toggled(bool)</signal>
   <receiver>otherLine</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>otherChx</sender>
   <signal>toggled(bool)</signal>
   <receiver>otherLine</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
